### PR TITLE
Adds new installation method 

### PR DESCRIPTION
**Pull request by vpetersson**
To be used either by `curl` or with directly from the repo.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+fi
+
+mkdir -p /opt/wott
+
+if [ ! -f /etc/wott/config.ini ]; then
+    echo -e "[DEFAULT]\\nenroll_token = $CLAIM_TOKEN" > /opt/wott/config.ini
+fi
+
+# If Ubuntu/Debian is detected
+if [ -f /etc/lsb-release ]; then
```
Comment (a-martynovich) [inline install.sh]: Will not work on Jessie and probably some other flavors. It's better to parse `/etc/os-release` instead, like we do this in Python code.

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+if [[ $(id -u) -ne 0 ]]; then
+    echo "Please run as root or using sudo"
+    exit 1
+fi
+
+mkdir -p /opt/wott
+
+if [ ! -f /etc/wott/config.ini ]; then
+    echo -e "[DEFAULT]\\nenroll_token = $CLAIM_TOKEN" > /opt/wott/config.ini
```
Comment (a-martynovich) [inline install.sh]: You're checking for `/etc/wott/config.ini` and then writing to `/opt/wott/config.ini`. Also where is `CLAIM_TOKEN` supposed to come from?

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+# -*- sh-basic-offset: 4 -*-
```
Comment (a-martynovich) [inline install.sh]: Is there any practical reason for setting *your* preferred Vim and Emacs settings in every bash script? Please don't tell me it's a "good practice".

File: install.sh (hunk 1 above)
Comment (vpetersson) [reply install.sh]: Do we need to check both, or is `/etc/os-release` sufficient for all?

File: install.sh (hunk 2 above)
Comment (vpetersson) [reply install.sh]: I'm checking to make sure it doesn't exist - and only if it *doesn't* exist, i write to it.

Claim token comes from an environment variable as mentioned in Slack.

Comment (a-martynovich) [reply install.sh]: Please read again. You're checking for `/etc/wott/config.ini` but writing to `/opt/wott/config.ini`.

File: install.sh (hunk 1 above)
Comment (a-martynovich) [reply install.sh]: `/etc/os-release` is sufficient for all.

File: install.sh (hunk 2 above)
Comment (vpetersson) [reply install.sh]: This is why we do peer reviews :)

File: bin/install.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    fi
+
+    mkdir -p /opt/wott
+
+    if [ ! -f /opt/wott/config.ini ]; then
+        echo -e "[DEFAULT]\\nenroll_token = $CLAIM_TOKEN" > /opt/wott/config.ini
+    fi
+
+    # If Ubuntu/Debian is detected
+    if [ -f /etc/os-release ]; then
```
Comment (a-martynovich) [inline bin/install.sh]: Not *test* it, *parse* it

Comment (vpetersson) [reply bin/install.sh]: Ok i've refactored it a fair bit now.

**Comment by vpetersson** [comment]
@a-martynovich this looks pretty solid now. Tested it across a number of distributions:

```
> $ for i in ubuntu:14.04 ubuntu:16.04 ubuntu:18.04 debian:8 debian:9 debian:10 centos:6 centos:7 centos:8 fedora; do docker run -v $(pwd):/app -e CLAIM_TOKEN=foobar --rm -ti $i /app/install.sh; done
Detected ubuntu
Detected ubuntu
Detected ubuntu
Detected debian
Detected debian
Detected debian
Unable to detect distribution. Exiting.
Detected CentOS. This is currently an unsupported distribution.
Detected CentOS. This is currently an unsupported distribution.
Detected Fedora. This is currently an unsupported distribution.
```

**Pull request merged**